www/src/components/button.js

```diff
@@ -34,6 +35,7 @@ const Button = ({
       ...(secondary && { ...buttonStyles.secondary }),
       ...(large && { ...buttonStyles.large }),
       ...(small && { ...buttonStyles.small }),
+      ...(tiny && buttonStyles.tiny),
```

Comment:
@greglobinski could you fix my crap above this line, too, please? 🙏😅

Reply:
sure :+1: